modules/apt.sh

```diff
@@ -61,26 +61,32 @@ _apt_write_list_file() {
     local repo_url="$2"
 
     cat >"$repo_file" <<EOF
-deb https://${repo_url}/ubuntu ${SERIES} main
-# deb-src https://${repo_url}/ubuntu ${SERIES} main
+deb ${repo_url}/ubuntu ${SERIES} main
+# deb-src ${repo_url}/ubuntu ${SERIES} main
 EOF
 }
 
 _apt_add_auth() {
-    local repo_host="$1"
+    local repo_path="$1"
```

Comment:
This is actually getting the hostname plus a path, just not the protocol prefix, right? `repo_path` doesn't sound like it would contain a hostname.

Since this branch is mostly about making sure the variable name (`repo_url`) matches what it contains, I think we should either rename `repo_path` to something like, ugh, `repo_host_path`, or even better perhaps, strip the protocol prefix inside this method (instead of in the caller) and call it `repo_url` again. Then we can strip the protocol further down in that echo call.

Care must be taken that we are only stripping https, not any other protocol, though, but the caller is making that assumption already.